README.md

```diff
@@ -55,7 +56,7 @@ packages.
 
  * Save and exit.
 
- * Run `make`
+ * Run `make` (for a real benchmarking setup do *not* use `-j`)
```

Comment:
FWIW, I only remembered about `-j` a few months ago when I saw laurie add support for it in `build.sh` in the warmup experiment :P

Let me know what you'd like to do.